src/Apps/Artist/Routes/AuctionResults/Components/AuctionResultHeader.tsx

```diff
@@ -1,21 +1,38 @@
-import { Box, Sans, Serif } from "@artsy/palette"
+import { Box, Link, Sans, Serif } from "@artsy/palette"
 import { AuctionResultHeader_artist } from "__generated__/AuctionResultHeader_artist.graphql"
 import React from "react"
 import { createFragmentContainer } from "react-relay"
 import { graphql } from "relay-runtime"
+import { data as sd } from "sharify"
 
 interface Props {
   artist: AuctionResultHeader_artist
 }
 
 const AuctionResultHeader: React.FC<Props> = props => {
   const { artist } = props
+
   return (
     <Box pb={2}>
       <Sans size="5t">Auction results</Sans>
       <Serif size="3" color="black100">
-        Filter past auction results to compare and evaluate {artist.name}'s
-        market.
+        Filter auction results to compare past lots by medium, size, and more.
+        Note that auction prices vary based on market specifics at the time of
+        the auction and may not be indicative of the current gallery market. To
+        get the best sense of value, pair the artist’s auction results with
+        their{" "}
+        <Link href={`${sd.APP_URL}/artist/${artist.slug}/cv`}>
+          career highlights
+        </Link>{" "}
```

Comment:
```suggestion
        <RouterLink href={`${sd.APP_URL}/artist/${artist.slug}/cv`}>
          career highlights
        </RouterLink>{" "}
```

Reply:
You'll need to add the import too

Reply:
Also, shouldn't need to include `sd.APP_URL`, but rather just the path: `/artist/${artist.slug}/cv`.

Reply:
That is great - Thank you both!! 🙏 will update